### PR TITLE
Pop ups

### PR DESCRIPTION
**Pull request by sntgopl**
-Created new branch for popUps.
-Created new div structure on html and css for popUps.
-Created js variables for each project button.
-Created function for multiple project buttons.
-Created array of objects for different projects.
-Add href to buttons inside project popUps.
-Created pull request.

### CODE REVIEW

File: style-desk.css [hunk 1]
```diff
@@ -680,4 +680,121 @@
       color: white;
     }
   }
+
+  #popUp {
```
Comment (sumairq) [inline style-desk.css:684]: - [ ] You design should be similar to Figma design in terms of sizing of elements in the popup. Let's make it similar to Figma design. 
- [ ] Your popup design should not have scroll in the description. Let's fix this issue. Cheers 👍 


Figma Design             |  Your Design
:-------------------------:|:-------------------------:
image  |  image

File: styles.css [hunk 2]
```diff
@@ -679,4 +703,98 @@
       opacity: 0;
     }
   }
+
+  #popUp {
```
Comment (sumairq) [inline styles.css:729]: - [ ] Your design is not fully responsive at mobile widths. Kindly make it more responsive. 
- [ ] I noticed that at mobile width the design stops being responsive altogether and sticks to the left of the screen. You can see the blank white area on the right. 
Let's fix these responsiveness issues. Cheers 👍  

Personal Portfolio (1)

**Review by sumairq — CHANGES_REQUESTED** [review]
## Hi Team ✋  , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔️ Interactive elements are working correctly
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @sntgopl , 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**